apps/systemtags/src/components/SystemTagsCreationControl.vue

```diff
@@ -64,7 +64,11 @@ export default {
 		handleResponse({ isRestricted, status, errorMessage, error }) {
 			if (status === 'ok') {
 				this.systemTagsCreationRestrictedToAdmin = isRestricted
-				showSuccess(t('settings', `System tag creation is now ${isRestricted ? 'restricted to administrators' : 'allowed for everybody'}`))
+				showSuccess(
+					t('settings', isRestricted
+						? 'System tag creation is now restricted to administrators'
+					 : 'System tag creation is now allowed for everybody'),
+				)
```

Comment:
This does not work. It will not be extracted correctly.

```suggestion
				showSuccess(isRestricted
					? t('settings', 'System tag creation is now restricted to administrators')
					: t('settings', 'System tag creation is now allowed for everybody'),
				)
```